core/translation.cpp

```diff
@@ -968,6 +968,19 @@ String TranslationServer::get_locale_name(const String &p_locale) const {
 	return locale_name_map[p_locale];
 }
 
+Array TranslationServer::get_all_loaded_locales() const {
```

Comment:
The "_all" in the name is probably redundant with the plural and the fact that the return value is an Array.

Reply:
I thought it might make it more intuitive, but I have made the change.